analysis/study_definition.py

```diff
@@ -1,168 +1,376 @@
+from datetime import date
 
-# Import functions
+from cohortextractor import StudyDefinition, patients
+import codelists
 
-from cohortextractor import (
-    StudyDefinition, 
-    patients, 
-    codelist, 
-    codelist_from_csv
-)
-
-# Import codelists
-
-chronic_cardiac_disease_codes = codelist_from_csv(
-    "codelists/opensafely-chronic-cardiac-disease.csv", system="ctv3", column="CTV3ID"
-)
-chronic_liver_disease_codes = codelist_from_csv(
-    "codelists/opensafely-chronic-liver-disease.csv", system="ctv3", column="CTV3ID"
-)
-salbutamol_codes = codelist_from_csv(
-    "codelists/opensafely-asthma-inhaler-salbutamol-medication.csv",
-    system="snomed",
-    column="id",
-)
-systolic_blood_pressure_codes = codelist(["2469."], system="ctv3")
-diastolic_blood_pressure_codes = codelist(["246A."], system="ctv3")
-
-
-# Specifiy study defeinition
 
 study = StudyDefinition(
-    # Configure the expectations framework
     default_expectations={
-        "date": {"earliest": "1900-01-01", "latest": "today"},
-        "rate": "exponential_increase",
+        "date": {"earliest": "1970-01-01", "latest": "today"},
+        "rate": "uniform",
+        "incidence": 0.1,
     },
-    # This line defines the study population
-    population=patients.registered_with_one_practice_between(
-        "2019-02-01", "2020-02-01"
-    ),
 
-    # https://github.com/opensafely/risk-factors-research/issues/49
+    population=patients.all(),
```

Comment:
limit to age 16-120?

Reply:
Have made this change.  Should we also filter to only include registered patients?

Reply:
aha good point, yes